gcp-cloud-run-fizzbuzz-example/src/_artifacts.tf

```diff
@@ -6,7 +6,7 @@ resource "massdriver_artifact" "endpoint" {
     {
       data = {
         api = {
-          hostname = google_cloud_run_service.main.status.0.url
+          hostname = trimprefix("${google_cloud_run_service.main.status.0.url}${var.api.path}", "https://")
```

Comment:
I don't think cloudrun needs the path here. Lambda does it because the lambda function is literally only listening on a single path. I think cloud run is expecting an entire API so the FE can add the path I think. @ChristensenJoe ?

Reply:
Cloud run needs the path or else it just returns a block of HTML (similar to azure function app)